src/cluster/k8sClientHandler.go

```diff
@@ -475,3 +475,25 @@ func GetNodesFromK8sClient() (*v1.NodeList, error) {
 	}
 	return nodeList, nil
 }
+
+func GetKubearmorRelayURL() string {
+	var namespace string
+	client := ConnectK8sClient()
+	if client == nil {
+		return ""
+	}
+
+	// get kubearmor pods from k8s api client
+	pods, err := client.CoreV1().Pods("").List(context.Background(), metav1.ListOptions{
+		LabelSelector: "kubearmor-app",
+	})
+	if err != nil {
+		log.Error().Msg(err.Error())
+		return ""
+	}
+	namespace = pods.Items[0].Namespace
+	url := "kubearmor." + namespace + ".svc.cluster.local"
```

Comment:
We need to return empty if the Pod List is  empty array as well,  which suggests there's no Kubearmor deployment active.